suse_migration_services/suse_connect.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2019 SUSE Linux LLC.  All rights reserved.
+#
+# This file is part of suse-migration-services.
+#
+# suse-migration-services is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# suse-migration-services is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with suse-migration-services. If not, see <http://www.gnu.org/licenses/>
+#
+# project
+from suse_migration_services.command import Command
+
+
+class SUSEConnect(object):
+    def is_registered(self):
+        status_command_result = Command.run(
+            ['SUSEConnect', '--list-extensions']
```

Comment:
How about capturing the output of SUSEConnect as the error message for the logs? (If the exit code is not zero, that is)

Reply:
yes the output of the SUSEConnect command should be logged in case of an error